setup.py

```diff
@@ -89,7 +91,7 @@
     "hiredis>=2.0.0,<3",
 ]
 
-AWS_REQUIRED = ["boto3>=1.17.0,<=1.20.23", "docker>=5.0.2", "s3fs>=0.4.0,<=2022.01.0"]
+AWS_REQUIRED = ["boto3>=1.24.11,<=1.24.95", "docker>=5.0.2", "s3fs>=0.4.0,<=2022.01.0"]
```

Comment:
there's a separate PR on #3585 . After that goes in, want to rebase? You'll also have to run `make lock-python-ci-dependencies PYTHON=x` and `make lock-python-dependencies PYTHON=x`

Reply:
The PR has landed now!